docs/Import-reference-genome.md

```diff
@@ -1,15 +1,15 @@
 # Add new reference genome to the database
 This manual is intended to guide you to load new reference genome(s) to the database. 
 
-### What is **Reference Genome** (also know as reference assembly)?
+### What is Reference Genome (also know as reference assembly)?
```

Comment:
`What is **Reference Genome** (also know as reference assembly)?` This section is not necessary for cBioPortal. Perhaps remove it?